src/cdk/scrolling/scrollable.ts

```diff
@@ -6,10 +6,35 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive, ElementRef, OnInit, OnDestroy, NgZone} from '@angular/core';
-import {Observable, Subject} from 'rxjs';
+import {Directionality} from '@angular/cdk/bidi';
+import {
+  getRtlScrollAxisType,
+  RtlScrollAxisType,
+  supportsScrollBehavior
+} from '@angular/cdk/platform';
+import {Directive, ElementRef, NgZone, OnDestroy, OnInit, Optional} from '@angular/core';
+import {fromEvent, Observable, Subject} from 'rxjs';
+import {takeUntil} from 'rxjs/operators';
 import {ScrollDispatcher} from './scroll-dispatcher';
 
+export type _Without<T> = { [P in keyof T]?: never };
+export type _XOR<T, U> = (_Without<T> & U) | (_Without<U> & T);
+export type _Top = { top?: number };
+export type _Bottom = { bottom?: number };
+export type _Left = { left?: number };
+export type _Right = { right?: number };
+export type _Start = { start?: number };
+export type _End = { end?: number };
```

Comment:
Omit spaces in braces

Reply:
We could probably make the new `noImportSpacing` rule also cover the spacing inside of objects. Similar as the [https://eslint.org/docs/rules/object-curly-spacing](object-curly-spacing) rule from ESlint.